tests/integration/components/bs-navbar-test.js

```diff
@@ -0,0 +1,83 @@
+import { moduleForComponent, test } from 'ember-qunit';
+import hbs from 'htmlbars-inline-precompile';
+
+moduleForComponent('bs-navbar', 'Integration | Component | bs-navbar', {
+  integration: true
+});
+
+test('it renders', function(assert) {
+  this.render(hbs`{{bs-navbar}}`);
+
+  assert.equal(this.$().text().trim(), '');
+
+  // Template block usage:
+  this.render(hbs`
+    {{#bs-navbar}}
+      template block text
+    {{/bs-navbar}}
+  `);
+
+  assert.equal(this.$().text().trim(), 'template block text');
+});
+
+test('it has correct markup', function(assert) {
+  this.render(hbs`{{bs-navbar}}`);
+
+  assert.equal(this.$('nav').length, 1, 'there is only one nav element');
+  assert.ok(this.$('nav').hasClass('navbar'), 'the navbar has the navbar class');
+  assert.ok(this.$('nav').hasClass('navbar-default'), 'the navbar has the navbar-default class');
+  assert.equal(this.$('nav > div').length, 1, 'there is a div right under the nav element');
+  assert.ok(this.$('nav > div').hasClass('container-fluid'), 'the div is a fluid container');
+});
+
+test('it handles inverse navbars properly', function(assert) {
+  this.render(hbs`{{bs-navbar type="inverse"}}`);
+
+  assert.ok(this.$('nav').hasClass('navbar-inverse'), 'the navbar has the navbar-inverse class');
+  assert.notOk(this.$('nav').hasClass('navbar-default'), 'the navbar does not have the navbar-default class');
+});
+
+test('it handles the toggling action propoerly', function(assert) {
```

Comment:
This test seems to be broken. First there is no `#clickme` element in your rendered template. This is not catched in the test probably because of the second error: your use of `assert.async` is wrong. It returns a callback that you have to call when your async stuff has finished. See https://api.qunitjs.com/async/ and https://github.com/kaliber5/ember-bootstrap/blob/fa66b53f862d2c893c06872fbe31c4a88d0cf567/tests/integration/components/bs-collapse-test.js#L31-L39 for examples.

This is probably also the cause of some other tests (progress bar) to fail, because your assertions here are run outside of the context of this test, as the test does not wait for your timeouts...